2_ThreeSum.cpp

```diff
@@ -62,31 +62,31 @@ using namespace std;
 vector<vector<int> > threeSum(vector<int> &num)
 {
 	//sort the array
-	sort(num.begin(), num.end());
+    sort(num.begin(), num.end());
     //ret contains the resultant triplets

```

Comment:
A well named variable makes comments like this redundant.  here if you had named it results or resultTriples or something similar, you could have omitted this comment

Reply:
Got it!